config.js

```diff
@@ -148,4 +148,10 @@ config.ngsi = {
 //clientSecret: 'IOTA_AUTH_CLIENT_SECRET'
 //};
 
+/**
+ * flag indicating whether the node server will be executed in multi-core option (true) or it will be a
+ * single-thread one (false).
+ */
+config.multiCore = true;
```

Comment:
Default in the repository should be false (or comment line, as I think you did or IOTA-JSON PR) in order to keep backward compatibility.

Reply:
Yes absolute, probably I left it for some of the tests.

Fixed in c6ab4db